versions.lock

```diff
@@ -174,11 +174,13 @@ org.apache.poi:poi:4.1.1 (1 constraints: 08050336)
 org.apache.poi:poi-ooxml:4.1.1 (1 constraints: 08050336)
 org.apache.poi:poi-ooxml-schemas:4.1.1 (1 constraints: 08050336)
 org.apache.poi:poi-scratchpad:4.1.1 (1 constraints: 08050336)
-org.apache.thrift:libthrift:0.13.0 (1 constraints: 8e0dfd33)
+org.apache.thrift:libthrift:0.14.1 (1 constraints: 900d0134)
 org.apache.tika:tika-core:1.24 (1 constraints: db04f730)
 org.apache.tika:tika-java7:1.24 (1 constraints: db04f730)
 org.apache.tika:tika-parsers:1.24 (1 constraints: db04f730)
 org.apache.tika:tika-xmp:1.24 (1 constraints: db04f730)
+org.apache.tomcat:tomcat-annotations-api:8.5.46 (1 constraints: 5a1162ea)
+org.apache.tomcat.embed:tomcat-embed-core:8.5.46 (1 constraints: 780c4b05)
```

Comment:
Would it be safe to exclude these two new tomcat dependencies or are they needed by our use of jaeger?

Reply:
I excluded them and tests still pass..